src/actions/actions.ts

```diff
@@ -962,6 +962,16 @@ class CommandEscInsertMode extends BaseCommand {
 
     vimState.currentMode = ModeName.Normal;
 
+    // If we wanted to repeat this insert, now is the time to do it. Insert
+    // count amount of these strings before returning back to normal mode
+    if (vimState.recordedState.count > 1) {
+      for (let i = 0; i < vimState.recordedState.count - 1; i++) {
+        vimState.recordedState.transformations.push({
```

Comment:
@rebornix instead of me trying to run "dot" here, is there a way to just push the recent content change multiple times?